### ISSUE TITLE
Replace method does not have an overload that supports 4 parameters.

### ISSUE THREAD

**Issue by mmccormick2211** [PowershellFrameworkCollective/PSModuleDevelopment#163]
Replace method does not have an overload that supports 4 parameters.  I have probably not referenced this correctly, but this error has caused complete malfunction of the Invoke-PSMDTemplate

_Originally posted by @mmccormick2211 in https://github.com/PowershellFrameworkCollective/PSModuleDevelopment/pull/155#r743224831_

**Comment by FriedrichWeinmann** [comment]
Heya thanks for reporting this, the latest version just shipped and solves the problem - templates now again working on Windows PowerShell 5.1
What happened?
I needed to switch from regex replacement to regular replacement, which is case sensitive.
I have mostly switch to PS7 by now, where .Replace has indeed up to 4 parameters, including a case-insensitive option.
Failed to test that on PS5.1, sorry :(
That said, I have now implemented my own custom replace method that solves that on all language versions.
